examples/finetune/README.md

```diff
@@ -0,0 +1,50 @@
+# finetune
+
+Basic usage instructions:
+
+```bash
+# get training data
+wget https://raw.githubusercontent.com/brunoklein99/deep-learning-notes/master/shakespeare.txt
+
+# finetune LORA adapter
+./bin/finetune \
+        --model-base open-llama-3b-v2-q8_0.gguf \
+        --checkpoint-in  chk-lora-open-llama-3b-v2-q8_0-shakespeare-LATEST.gguf \
+        --checkpoint-out chk-lora-open-llama-3b-v2-q8_0-shakespeare-ITERATION.gguf \
+        --model-out lora-open-llama-3b-v2-q8_0-shakespeare-ITERATION.bin \
+        --train-data "shakespeare.txt" \
+        --save-every 10 \
+        --threads 6 --adam-iter 30 --batch 4 --ctx 64 \
+        --use-checkpointing --use-alloc \
+        --mem-lora 2 --mem-compute 1 --mem-compute0 20
+
+# predict
+./bin/main -m open-llama-3b-v2-q8_0.gguf --lora lora-open-llama-3b-v2-q8_0-shakespeare-LATEST.bin
```

Comment:
.bin -> .gguf

Reply:
'.bin' is correct here, because the LORA adapters itself are not yet in GGUF format. Llama.cpp still uses the binaray 'ggla' format for them, so did I when exporting.

Reply:
oh. yea i miss read.
`--model-out lora-open-llama-3b-v2-q8_0-shakespeare-ITERATION.bin`